app/controllers/AdditionalMetadataController.scala

```diff
@@ -16,12 +16,24 @@ class AdditionalMetadataController @Inject() (
     val controllerComponents: SecurityComponents
 ) extends TokenSecurity {
 
+  val metadataProperties: Map[String, List[String]] = Map(
```

Comment:
Could these values be added to the display properties table, if they are not there already?

Reply:
Yeah, we can add them for closure and descriptive metadata types.
Not sure about the name of the attribute.

 PropertyName |   Attribute       |                 Value           | AttributeType 
--------------+---------------+---------------------------------------+---------------
 ClosurePeriod  | PropertyInfo   | Closure period             | text

Is that fine?

Reply:
Can you ask Phil and Julius about this, but I'm happy with the `PropertyInfo` attribute name.